variod.c

```diff
@@ -361,7 +369,7 @@ int main(int argc, char *argv[])
 		fprintf(fp_console, "Connection dropped\n");	
 		fflush(fp_console);
 		
-		close(xcsoar_sock);
+		//close(xcsoar_sock);
```

Comment:
I think it is still a good idea to close the socket on exit.

Reply:
Yes, I'll move it out of the while(1) loop so it doesn't close the socket each time.
If connection drops, the reconnection sequence closes it first.